src/server/types.ts

```diff
@@ -71,10 +71,17 @@ export interface RouteConfig {
   csp?: boolean;
 }
 
+export interface RenderOptions {
+  statusOverwrite?: number;
```

Comment:
```suggestion
  status?: number;
  statusText?: string;
```

Let's add this option too while we're at it.